src/stories/src/components/expansion-panel/expansion-panel.stories.tsx

```diff
@@ -21,7 +21,7 @@ export const Default: Story<IExpansionPanelProps> = ({
   useAnimations = true
 }) => (
   <ForgeExpansionPanel open={open} orientation={orientation} useAnimations={useAnimations} style={{ width: '250px' }}>
-    <div role="button" tabIndex={0} slot="header">Click me</div>
+    <button slot="header" style={{ all: 'unset' }}>Click me</button>
```

Comment:
Should we update this example with manual handling of the accessibility attributes?